### ISSUE TITLE
Difficulty Setting

### ISSUE THREAD

**Issue by JacobSchmitt** [omnilingo/omnilingo#61]
option for number of unknown words/blanks per card (2 instead of 1)
      basically increase difficulty until you're writing whole sentences

**Comment by d33tah** [comment]
IMO difficulty customization is such a big feature that we need to come up with a separate UX for it. It's already beyond one or two sliders. The way I see it is a button "Set difficulty" that shows a modal window, allowing the user to either select a preset or tweak their preferences.

**Comment by d33tah** [comment]
Also @JacobSchmitt, isn't it a bit of a duplicate of #54 I feel like we need to group those issues together and set up a shared milestone for them.

CCing @ftyers

**Comment by ftyers** [comment]
I don't think this is a duplicate, but it is a very big issue, and the answer will likely be different depending on the task.

e.g. for the scramble task, an "easy" mode would be to give the exact number of letters instead of just the set of them.

In terms of UX i'd say a first version should have a simple UX defined by us, with maybe customisation hidden behind a modal. We also need to think about the semantics of "levels" and "difficulty". E.g. I see at least two axes here:
1. Language level (difficulty of structures and words)
2. Task level (difficulty of task, how many gaps, how many options, how close are the distractors, etc.)
